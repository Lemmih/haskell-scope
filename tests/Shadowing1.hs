module Shadowing1 where

x = \x -> x
