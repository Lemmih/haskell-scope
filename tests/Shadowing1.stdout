Scope errors:

Definitions:
  Definition 1, from Shadowing1.decl0:
    [4mx[0m = \x -> x
  Definition 2, from Shadowing1.decl0.rhs.lambda:
    x = \[4mx[0m -> x

Use sites:
  Definition used: 2
    x = \x -> [4mx[0m
